drakrun/drakrun/analyzer.py

```diff
@@ -0,0 +1,515 @@
+        )
+
+        drakmon_log_path = output_dir / "drakmon.log"
+        with run_tcpdump(vm, options), drakmon_log_path.open("wb") as drakmon_log:
+            try:
+                subprocess.run(
+                    drakvuf_cmd,
+                    stdout=drakmon_log,
+                    check=True,
+                    timeout=options.timeout + 60,
+                )
+            except subprocess.CalledProcessError as e:
+                # see DRAKVUF src/exitcodes.h for more details
+                INJECTION_UNSUCCESSFUL = 4
+
+                if e.returncode == INJECTION_UNSUCCESSFUL:
+                    log.error("Sample startup failed")
+                else:
+                    # Something bad happened
+                    raise e
+            except subprocess.TimeoutExpired as e:
+                log.error("DRAKVUF timeout expired")
+                raise e
+
+        if options.raw_memory_dump:
+            vm.vm.memory_dump(str(output_dir / "post_sample.raw_memdump.gz"))
+
+    log.info("Analysis done. Collecting artifacts...")
+
+    # Make sure dumps have a reasonable size.
+    # Calculate dumps_metadata as it's required by the `analysis` task format.
+    dumps_path = output_dir / "dumps"
+    dumps_zip_path = output_dir / "dumps.zip"
+    dumps_metadata = crop_dumps(str(dumps_path), str(dumps_zip_path))
+
+    # Compress IPT traces, they're quite large however they compress well
+    ipt_path = output_dir / "ipt"
+    ipt_zip_path = output_dir / "ipt.zip"
+    compress_ipt(str(ipt_path), str(ipt_zip_path))
+
+    time_finished = time.time()
+    return {
+        "time_started": time_started,
+        "time_finished": time_finished,
+        "start_command": start_command,
+        "dumps_metadata": dumps_metadata,
+    }
+
+
+class PluginsArgAction(argparse.Action):
+    def __init__(self, option_strings, dest, nargs=None, **kwargs):
+        if nargs is not None:
+            raise ValueError("nargs not allowed")
+        super().__init__(option_strings, dest, **kwargs)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        if option_string is None:
+            raise ValueError("Missing --plugins value")
+        setattr(namespace, self.dest, option_string.split(","))
+
+
+def main():
+    logging.basicConfig(level=logging.INFO)
+    drakconfig = load_config()
+    parser = argparse.ArgumentParser(
+        description="Analyze file in Drakvuf",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument("sample_path", help="Path to the sample")
+    parser.add_argument(
+        "--vm-id",
+        help="Virtual machine ID to use for analysis",
+        type=int,
+        required=True,
+    )
+    parser.add_argument(
+        "--output-dir",
+        help="Path where analysis results will be written",
+        required=True,
+    )
+    parser.add_argument(
+        "--plugins",
+        help="DRAKVUF plugins to enable during analysis",
```

Comment:
```suggestion
        help="Comma separated DRAKVUF plugins to use",
```
(I assume they're comma separated)